src/content-scripts/components/viewer/ViewerManager.tsx

```diff
@@ -6,40 +6,38 @@ import _isEqual from 'lodash/isEqual';
 
 import { hideViewer, setMouseOverViewer } from 'common/store/tabs/tab/widgets/actions';
 import { selectViewerState } from 'common/store/tabs/tab/widgets/selectors';
-import { PPViewerHoverContainerClass } from 'content-scripts/settings';
 
 import Viewer from './Viewer';
 
+interface IViewerManagerProps {
+  visible: boolean;
+  mouseOver: boolean;
+  isDeleteModalOpen: boolean;
+  annotationIds: string[];
+  isAnyReportEditorOpen: boolean;
+
+  hideViewer: () => void;
+  setMouseOverViewer: (value: boolean) => void;
+}
+
 interface IViewerManagerState {
-  isMouseOver: boolean;
```

Comment:
We have now plenty of other variables starting with `is...`, so are you sure you want to make an exception?

I thought about this is/has convention for a second if it is only me liking it or it is maybe broader standard (still I don't mean _universal_). So I googled and it looks like the latter, for example there is event [eslint package](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming.md) for that with 5k stars :smiley: so

Reply:
I just wanted to be consistent (with `mouseOver` elsewhere), but we might make it consistent also the other way around :)

Reply:
Oh, OK. I would prefer it this way :)

 We could also see how far we can propagate this convention.

Reply:
Yeah, it might be fifty-fifty now for all I know :)